### PR TITLE
add attribution for new basemap_2016 with carto tiles and osm data

### LINKED ISSUE #1537: attribution
add a control to the map to specify the attribution of a layer. this is necessary for the new roadmap which inludes OSM data

### PR DESCRIPTION
**Pull request by petzlux**
fix #1537

### CODE REVIEW

**Comment by petzlux** [comment]
screen shot 2016-11-03 at 16 13 50

**Comment by jaykayone** [comment]
LGTM, @rmichaelis can you please review ?

File: geoportailv3/static/js/themesservice.js [hunk 1]
```diff
@@ -125,6 +125,13 @@ app.Themes.prototype.getBgLayers = function() {
           goog.asserts.assert('imageType' in item);
           var layer = this.getWmtsLayer_(item['name'], item['imageType']);
           layer.set('metadata', item['metadata']);
+
+          if (goog.object.containsKey(item['metadata'], 'attribution')) {
+            var source = layer.getSource();
+            source.setAttributions(
+              item['metadata']['attribution']
```
Comment (rmichaelis) [inline geoportailv3/static/js/themesservice.js]: Do you know how OL3 reacts if a same text is provided to different layers ? Is the same text displayed displayed many times ?

Comment (petzlux) [inline geoportailv3/static/js/themesservice.js]: yes multiple attributions are displayed on multiple lines, no intelligence  , but thats expected.

**Comment by petzlux** [comment]
LGTM ! @jaykayone

**Pull request merged**